src/TextBoxGroup/text-box-group-style.scss

```diff
@@ -16,37 +18,44 @@ $gray-500: #adadad;
 $transition-time: 200ms;
 .form-group.input-box-group {
     position: relative;
-    >label.custom-label:not(.form-check-label):not(.custom-checkbox):not(.custom-radio):not(.custom-control-label) {
-        display: block;
-        margin-bottom: 4px;
-        line-height: 1;
-        font-size: 12px;
-        font-weight: normal;
-    }
-    >.input-group {
+    > .input-group {
         position: relative;
-        >.input-box-group-wrapper {
+        > .input-box-group-wrapper {
+            color: $text;
+            will-change: border;
```

Comment:
`will-change` is used with components that will be changed 100% and it might be expensive to the browser to execute its transitions. It is not recommended to be used with components that barely changes or the transitions are minimal. Using it might cause performance issues. [Read this](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change).

Reply:
resolved in commit https://github.com/sebgroup/react-components/pull/29/commits/2b668852b23d63790e2f37e7a82001a8673b8ad5